includes/Services/FlowService.php

```diff
@@ -0,0 +1,203 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\Services;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Data\Flows;
+use NewfoldLabs\WP\Module\Onboarding\Data\Data;
+
+class FlowService {
+
+	public static function initalize_flow_data() { 
+		$default_flow_data =  self::get_default_flow_data();
+         if ( ! ( $flow_data = self::read_flow_data_from_wp_option() ) ) {
+			return \update_option( Options::get_option_name( 'flow' ), $default_flow_data );
+		}
+		$updated_flow_data = self::update_flow_data_recursive($default_flow_data, $flow_data);
+		return \update_option( Options::get_option_name( 'flow' ), $updated_flow_data );
+    }
+
+    public static function get_default_flow_data() {
+		// check if data is available in the database if not then fetch the default dataset
+		$flow_data              = Flows::get_data();
+		$flow_data['createdAt'] = time();
+		// get current flow type
+		$flow_type = Data::current_flow();
+		// update default data if flow type is ecommerce
+		if($flow_type == 'ecommerce') 
+			$flow_data = self::update_default_data_for_ecommerce( $flow_data );
+		return $flow_data;
+	}
+
+	public static function update_flow_data($params) {
```

Comment:
```suggestion
	public static function get_updated_flow_data($params) {
```
Since it is not updating the DB